### PR TITLE
rofi-emoji: update to 3.0.0.

### PR DESCRIPTION
**Pull request by motorto**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

Sorry for the previous pr included work from the other branch ...

### CODE REVIEW

File: srcpkgs/rofi-emoji/template [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Template file for 'rofi-emoji'
 pkgname=rofi-emoji
-version=2.3.0
+version=3.0.0
 revision=1
 build_style=gnu-configure
-hostmakedepends="autoconf automake libtool pkg-config"
+hostmakedepends="autoconf automake libtool pkgconf"
```
Comment (classabbyamp) [inline srcpkgs/rofi-emoji/template:6]: why switch?

Comment (motorto) [reply srcpkgs/rofi-emoji/template:6]: some macro https://github.com/Mange/rofi-emoji/issues/45

Comment (classabbyamp) [reply srcpkgs/rofi-emoji/template:6]: looks like 3.0.1 fixes it :)

Comment (motorto) [reply srcpkgs/rofi-emoji/template]: Updated to it !

**Pull request merged**